src/contracts/proxies/ODProxy.sol

```diff
@@ -19,4 +26,24 @@ contract ODProxy is Ownable {
       revert TargetCallFailed(_response);
     }
   }
+
+  function setOwner(address _owner) external override {
+    require(msg.sender == _VAULT721, 'Only Vault721');
```

Comment:
what situation does the vault721 contract ever need to transfer this? it would only be for a batch transfer of all the nfts. I'm not convinced this is the best way to do this, and it makes the proxy not transferable by other downstream parts of the protocol